test/Npgsql.Tests/CommandTests.cs

```diff
@@ -828,6 +828,33 @@ public async Task Use_across_connection_change([Values(PrepareOrNot.Prepared, Pr
         Assert.That(await cmd.ExecuteScalarAsync(), Is.EqualTo(1));
     }
 
+#if DEBUG
```

Comment:
Can still run this test in RELEASE, no harm in that (and may trip future bugs or whatever).

Reply:
Not sure I really see the point in it but I've enabled it.